lib/cnet/pcb/cnet_pcb.c

```diff
@@ -191,15 +191,15 @@ pcb_show(mempool_t *mp __cne_unused, void *obj_cb_arg __cne_unused, void *obj,
     cne_printf("[green]%-6s [orange] %04x [red]%6s[]", pcb->closed ? "Closed" : "Open",
                pcb->opt_flag, chnl_protocol_str(pcb->ip_proto));
 
-    if (snprintf(fbuf, sizeof(fbuf), "%s:%d",
-                 inet_ntop4(ip1, sizeof(ip1), &pcb->key.faddr.cin_addr, NULL),
+    b1 = inet_ntop4(ip1, sizeof(ip1), &pcb->key.faddr.cin_addr, NULL);
+    if (snprintf(fbuf, sizeof(fbuf), "%s:%d", (b1) ? b1 : "Invalid IP",
```

Comment:
You don't need the parenthesis around the (b1).

You also don't need two variables, b1 and b2. And actually you don't need both fbuf and lbuf and ip1 and ip2.
Could be simpler, IMO.

```
    char buf[128], ipb[IP4_ADDR_STRLEN] = {0}, *ip;
    uint16_t port;

    ip = inet_ntop4(ipb, sizeof(ipb), &pcb->key.faddr.cin_addr, NULL);
    port = ntohs(CIN_PORT(&pcb->key.faddr));

    if (snprintf(buf, sizeof(buf), "%s:%d", ip ?: "Invalid IP", port) < 0)
        CNE_RET("Truncated buffer data\n");
    cne_printf(" [orange]%20s[]", buf);

    ip = inet_ntop4(ipb, sizeof(ipb), &pcb->key.laddr.cin_addr, NULL);
    port = ntohs(CIN_PORT(&pcb->key.laddr));

    if (snprintf(buf, sizeof(buf), "%s:%d", ip ?: "Invalid IP", port) < 0)
        CNE_RET("Truncated buffer data\n");
    cne_printf(" [orange]%20s[]", buf);
```

Reply:
will remove the parenthesis and use one variable